app/components/registryBrowseTags/registryBrowseTags.html

```diff
@@ -0,0 +1,44 @@
+<rd-header>
+  <rd-header-title title="Repository Tags"></rd-header-title>
+  <rd-header-content>
+    <a ui-sref="registries">Registries</a> &gt; <a ui-sref="registry.browse({id: registry.Id})">{{ registry.Name }}</a> &gt; <a ui-sref="registry.tags({id: registry.Id, repository: repository})">{{ repository }}</a>
+  </rd-header-content>
+</rd-header>
+
+<!--<div class="row" ng-if="tags">
```

Comment:
What is this section supposed to display ? Should we keep it ? If no, please remove the comments.